buildSrc/win.gradle

```diff
@@ -41,6 +41,10 @@ WIN.modLibDest = "lib"
 
 def CPU_BITS = IS_64 ? "x64" : "x86"
 
+def HOST_ARCH = getWinArch(ext.OS_ARCH)
+def TARGET_ARCH = getWinArch(ext.TARGET_ARCH)
+def IS_CROSS = HOST_ARCH != TARGET_ARCH
```

Comment:
can we move this to build.gradle?
With the increasing number of supported configurations, there will otherwise be more gradle sub-files that have something similar.

Reply:
Yes, this does seem like a better plan. Should this be done as a follow-on or do you want to see it done now? One reason I ask is that my PR #462 (which is now approved, but waiting re-review) does something similar to `getWinArch()` to determine whether we are running on aarch64 or not and should be modified as well.

Reply:
In that case, a follow-up seems best (we can tackle mac/win/linux aarch64 simultaneously then)

Reply:
I added CONVERTED_OS_ARCH, CONVERTED_TARGET_ARCH variables, and getConvertedOsArch(arch) function to build.gradle.

I think there should be the better prefix name for updated os arch but what comes in my mind is converted or unified.

Building jfx with the fix on windows 32, 64 bits and arm64 cross compilation outputs the following variables in the log:
```
# Win 32 bits
OS_ARCH: x86
TARGET_ARCH: x86
CONVERTED_OS_ARCH: x86
CONVERTED_TARGET_ARCH: x86

# Win 64 bits
OS_ARCH: amd64
TARGET_ARCH: amd64
CONVERTED_OS_ARCH: x64
CONVERTED_TARGET_ARCH: x64

# Win arm64 cross compilation
OS_ARCH: amd64
TARGET_ARCH: aarch64
CONVERTED_OS_ARCH: x64
CONVERTED_TARGET_ARCH: arm64

```